clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -3796,6 +3796,13 @@ public DescribeShareGroupsResult describeShareGroups(final Collection<String> gr
                 .collect(Collectors.toMap(entry -> entry.getKey().idValue, Map.Entry::getValue)));
     }
 
+    // To do in a follow-up PR
+    @Override
+    public ListShareGroupOffsetsResult listShareGroupOffsets(final Map<String, ListShareGroupOffsetsSpec> groupSpecs,
+                                                             final ListShareGroupOffsetsOptions options) {
+        // To-do
+        return null;
```

Comment:
nit: generally better to throw Exception that not yet implemented but it's fine for now as follow up PRs will be soon after.